app/controllers/state_file/intake_logins_controller.rb

```diff
@@ -75,7 +76,25 @@ def service_type
     end
 
     def redirect_to_data_review_if_intake_authenticated
```

Comment:
should we rename this to reflect the fact that they can be redirected to different places now?